physionet-django/user/views.py

```diff
@@ -438,16 +440,17 @@ def public_profile(request, username):
         'public_email':public_email, 'projects':projects})
 
 
-def profile_photo(request, username):
+def profile_photo(_request, username):
     """
     Serve a user's profile photo
     """
     try:
         user = User.objects.get(username__iexact=username)
-        return utility.serve_file(user.profile.photo.path)
     except ObjectDoesNotExist:
         raise Http404()
 
+    return UserFiles().serve_photo(user)
```

Comment:
Doesn't this change the logic when the photo is missing?

Reply:
nope, it looks that everything is as before